src/nodes.jl

```diff
@@ -329,7 +329,7 @@ set_attribute(x::XMLElement, name::AbstractString, val) = set_attribute(x, name,
 if VERSION < v"0.4.0-dev+980"
     const PairTypes = NTuple{2}
 else
-    const PairTypes = Union(NTuple{2}, Pair)
+    const PairTypes = Union{NTuple{2}, Pair}
```

Comment:
there may be a range of 0.4.0-dev versions after 980 that don't understand Union syntax with curly braces where this would break. Maybe not highest priority to support but worth checking.